src/pseudopeople/entity_types.py

```diff
@@ -21,15 +21,18 @@ class RowNoiseType:
     """
 
     name: str
-    noise_function: Callable[[pd.DataFrame, float, RandomnessStream], pd.DataFrame]
+    noise_function: Callable[[pd.DataFrame, float, RandomnessStream, str], pd.DataFrame]
 
     def __call__(
         self,
         form_data: pd.DataFrame,
         configuration: float,
         randomness_stream: RandomnessStream,
+        additional_key: str,
```

Comment:
Do we need an additional key for row noise?  It would only look at the dataframe once and pick which rows to noise.